shells/web-shell/elements/web-config.js

```diff
@@ -56,12 +56,11 @@ export class WebConfig extends Xen.Debug(Xen.Async, log) {
       userid: params.get('user') || localStorage.getItem(Const.LOCALSTORAGE.user),
       arckey: params.get('arc'),
       search: params.get('search') || '',
-      storageKeyBase: params.get('plannerStorageKeyBase') || params.get('storageKeyBase') || 'volatile://',
-      debug: params.get('planificatorDebug'),
-      onlyConsumer: params.get('onlyConsumer') === 'true',
+      plannerStorage: params.get('plannerStorage') || params.get('plannerStorageKeyBase') || params.get('storageKeyBase') || 'volatile://',
```

Comment:
maybe just leave one for every param?

Reply:
Until some time has passed and everybody has the right info, I thought it best to leave the old ones for backward compatibility.